.nuget/NuGet.config

```diff
@@ -1,14 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <solution>
-    <add key="disableSourceControlIntegration" value="true" />
```

Comment:
are there specific reasons for removing `disableSourceControlIntegration` setting?

Reply:
This consolidates the set of package references (generally) down into the root directory, and hence the nuget.config is there.  This particular key would now be generally irrelevant because packages should restore into the "nuget cache" folder on disk, but also because of the .gitignore indicating to ignore the packages folder in all cases.  https://github.com/dotnet/ef6tools/blob/3b5c5620a664172558b29b5e34377d342af3a1a6/.gitignore#L15